src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/FileWriter.java

```diff
@@ -105,6 +106,10 @@ void rotate() throws IOException {
     }
 
     void finishFile() throws IOException {
```

Comment:
I don't see the value of this empty overload, it just hides the modifier, which is kinda important at this point.